### DISCUSSION TITLE
Discrepancy between the Expanded Entity Structure as described in the docs and generated code

### DISCUSSION THREAD

**Discussion by JonasCir** [SeaQL/sea-orm#1879] [General]
## Description

Hey there,
I noticed that `ActiveModel` is not visible in the generated code when using the `--expanded-format` option.  This leads to a discrepancy between the Expanded Entity Structure as described in the official [docs](https://www.sea-ql.org/SeaORM/docs/generate-entity/expanded-entity-structure/) and the actual generated result.

## Steps to Reproduce

Run the expanded code generation on any of the examples in this repository.

### Expected Behavior

The `ActiveModel` struct should be visible in the expanded generated code.

### Actual Behavior

You will notice that the generated code is missing the  `ActiveModel` struct when using the `--expanded-format` option

### Reproduces How Often

Always

### Workarounds

None

## Reproducible Example

`sea-orm-cli generate entity --expanded-format -o exp_entity` in `sea-orm/examples/actix_example`

## Versions

`sea-orm-cli 0.11.3` on Linux and SQLite DB

**Comment by tyt2y3** [comment]
Expanded entity structure is really a legacy and there is little motivation to change it at this point.
It used to exist because IntelliJ cannot expand derive macros, and it would lead to lots of missing symbols. I think this has been solved already.
